README.md

```diff
@@ -1,99 +1,9 @@
-  - [`lit-html`](./packages/lit-html) - The rendering library used by LitElement.
-  - [`@lit/reactive-element`](./packages/reactive-element) - A low level base class that provides a reactive lifecycle based on attribute/property changes.
-- Additional libraries
-  - [`@lit/localize`](./packages/localize) - A library and command-line tool for localizing web applications built using Lit.
-  - [`@lit/localize-tools`](./packages/localize-tools) - Localization tooling for use with `@lit/localize`.
-- Labs
-  - [`@lit-labs/ssr`](./packages/labs/ssr) - A server package for rendering Lit templates and components on the server.
-  - [`@lit-labs/ssr-client`](./packages/labs/ssr-client) - A set of client-side support modules for rendering Lit components and templates on the server using `@lit-labs/ssr`.
-  - [`@lit-labs/eleventy-plugin-lit`](./packages/labs/eleventy-plugin-lit) - A plugin for Eleventy that pre-renders
-    Lit components using `@lit-labs/ssr` with optional hydration.
-  - [`@lit-labs/react`](./packages/labs/react) - A React component wrapper for web components.
-  - [`@lit-labs/router`](./packages/labs/router) - A router for Lit.
-  - [`@lit-labs/task`](./packages/labs/task) - A controller for Lit that renders asynchronous tasks.
-  - [`@lit-labs/motion`](./packages/labs/motion) - Lit directives for making things move
-  - [`@lit-labs/scoped-registry-mixin`](./packages/labs/scoped-registry-mixin) - A mixin for LitElement that integrates with the speculative Scoped CustomElementRegistry polyfill.
-- Starter kits (not published to npm)
-  - [`lit-starter-ts`](./packages/lit-starter-ts) ([template
-    repo](https://github.com/lit/lit-element-starter-ts)) - A starter repo for building reusable components using Lit in TypeScript.
-  - [`lit-starter-js`](./packages/lit-starter-js) ([template
-    repo](https://github.com/lit/lit-element-starter-js)) - A starter repo for building reusable components using Lit in Javascript.
-- Internal packages (not published to npm)
-  - [`tests`](./packages/tests) - Test infrastructure for the monorepo.
-  - [`benchmarks`](./packages/benchmarks) - Benchmarks for testing various libraries in the monorepo.
-  - [`@lit-internal/scripts`](./packages/@lit-internal/scripts) - Utility scripts used within the monorepo.
-
-## Contributing to Lit
-
-Lit is open source and we appreciate issue reports and pull requests. See [CONTRIBUTING.md](./CONTRIBUTING.md) for more information.
-
-### Setting up the lit monorepo for development
-
-Initialize repo:
-
-```sh
-git clone https://github.com/lit/lit.git
-cd lit
-npm ci
-```
-
-Build all packages:
-
-```sh
-npm run build
-```
-
-Test all packages:
-
-```sh
-npm run test
-```
-
-Run benchmarks for all packages:
-
-```sh
-npm run benchmarks
-```
-
-See individual package READMEs for details on developing for a specific package.
+In `rollup-common.js` option `mangle` in `generateTerserOptions` is disabled.
```

Comment:
There are three notable changes:

1. No mangling.
2. Packages are renamed.
3. Dependencies are changed to forks. 

Point 3 was needed since we did not manage to override them at the external-board-tools side.